Formula/crfsuite.rb

```diff
@@ -0,0 +1,41 @@
+class Crfsuite < Formula
+  desc "Fast implementation of conditional random fields"
+  homepage "https://www.chokkan.org/software/crfsuite/"
+  url "https://github.com/downloads/chokkan/crfsuite/crfsuite-0.12.tar.gz"
```

Comment:
Wow, I didn't know there's such links like `https://github.com/downloads/...`

I'd suggest switching it to the format we usually use:

```suggestion
  url "https://github.com/chokkan/crfsuite/archive/refs/tags/0.12.tar.gz"
```

Reply:
Good catch. This is borrowing from an existing formula in the `homebrew-science` (legacy) tap.

Reply:
Actually, these two tarballs don't have the same contents. The original one I had includes a pre-generated `configure` file. The `/archive` tarball doesn't have it.

It seems there are other formulae in core tap that use this kind of link, e.g. `apktool`, `jadx`.

Reply:
Oh, wow, old stuff!
TIL there was a downloads API that was replaced with releases like 10 years ago.

Source: https://stackoverflow.com/questions/63292793/how-can-i-upload-files-to-https-github-com-downloads-user-repo